packages/epo-react-lib/src/components/DemoButton/DemoButton.tsx

```diff
@@ -3,17 +3,17 @@ import styled from "styled-components";
 const DemoButton = styled.button`
   border: none;
   border-radius: 0.5rem;
-  background-color: #186faf;
+  background-color: var(--turquoise80, black);
```

Comment:
use some of our global CSS vars to verify that the styled-components global styles are being added to storybook